tests/test_switch_repo.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_switch_repo(base_url, selenium):
+        """ Switch to new active watched repo """
```

Comment:
After making both the comment and the code indented four spaces all errors resolved.